flow/envs/green_wave_env.py

```diff
@@ -468,8 +473,10 @@ def __init__(self, env_params, sumo_params, scenario):
     @property
     def observation_space(self):
         """
-        Partial observed state space. Velocities, distance to intersections,
-        edge number (for nearby vehicles) traffic light state
+        Partial observed state space.
```

Comment:
Super nitpicky, but should this be "Partially observed"?

Reply:
Not too nitpicky, this is a good catch.